pol-core/bscript/compiler/astbuilder/ExpressionBuilder.cpp

```diff
@@ -431,8 +432,9 @@ std::unique_ptr<FunctionCall> ExpressionBuilder::function_call(
 {
   auto arguments = value_arguments( ctx->expressionList() );
 
+  // Expression-as-callee functions, eg. `(variable)(args...)` do not have a call scope or name.
```

Comment:
true, but this comment does not relate to the code anymore, since FunctionCall now has two ctors vs one that handles both types of calls. remove comment